common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1401,7 +1402,7 @@ public enum API {
             MSAL_HELLO(MSAL_HELLO_PATH, null, VERSION_3),
             ACQUIRE_TOKEN_INTERACTIVE(MSAL_ACQUIRE_TOKEN_INTERACTIVE_PATH, null, VERSION_3),
             ACQUIRE_TOKEN_SILENT(MSAL_ACQUIRE_TOKEN_SILENT_PATH, null, VERSION_3),
-            GET_ACCOUNTS(MSAL_GET_ACCOUNTS_PATH, null, VERSION_3),
+            GET_ACCOUNTS(MSAL_GET_ACCOUNTS_PATH, null, VERSION_8),
```

Comment:
This shouldn't be 8. 

Only apps that requires WPJ account from PRT needs to set to 8.

Reply:
From my understanding, if we want to increment the msal-broker protocol version in broker we need to increment the version here, because the version is calculated from the max number in the BrokerContentProvider.API
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/b8ea54b8ea2e37f7cc40939752a6dd582c6ae21c/common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java#L599
hence the change here in the GET_ACCOUNTS API.

still old msal versions will work, see:
https://github.com/AzureAD/ad-accounts-for-android/blob/43aa86750ea846015e4c73a50f05a08cdbe5296a/AADAuthenticator/src/msal/java/com/microsoft/identity/client/BrokerUtils.java#L96